run_http_server_dynamodb.sh

```
@@ -0,0 +1,257 @@
+
+echo "Schema migration is disabled. Schema service will start with an empty database."
+
+# Run the HTTP server in the background with schema service URL
+echo "Starting the HTTP server on port 9001 with DynamoDB backend..."
+echo "Note: DynamoDB tables will be created automatically if they don't exist."
+echo "Make sure AWS credentials are configured (AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, or IAM role)"
+
+# Debug: Print AWS Credential Status
+
+# Export OPENROUTER_API_KEY if set in .zshrc
+source ~/.zshrc 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)